wazuh/wazuh-dashboard-plugins
# Fix the filter are displayed cropped on screens of 575px to 767px

### Description

Fix the filter are displayed cropped on screens of 575px to 767px
 
### Issues Resolved

#7040 

### Evidence

<img width="1792" alt="Captura de pantalla 2024-10-01 a la(s) 3 27 07 p  m" src="https://github.com/user-attachments/assets/e20768c9-c6bd-4d84-bed5-5bce82400748">
<img width="1792" alt="Captura de pantalla 2024-10-01 a la(s) 3 14 13 p  m" src="https://github.com/user-attachments/assets/6da33757-62ca-49ff-87e2-9fdbfe7af21e">
<img width="1792" alt="Captura de pantalla 2024-10-01 a la(s) 3 14 00 p  m" src="https://github.com/user-attachments/assets/a87e5df3-5084-42ec-8377-2e446d837e49">

### Test

- Go to the vulnerability detection module

- Add a long filter

- Check that the filter is not cut off on screens of 575px to 767px

### Check List
